### ISSUE TITLE
Gradient accumulation example on WMT

### ISSUE THREAD

**Issue by AkashGanesan** [google/flax#827]
Hey, we'd like to accumulate gradients.  Is there a barebones example (for  multi-GPU training) for this that we can follow? 

### Description of the model to be implemented
Transformer-base.  Potentially other transformer models as well.

### Dataset the model could be trained on
WMT.*

### Reference implementations in other frameworks
https://stackoverflow.com/questions/53331540/accumulating-gradients

**Comment by andsteing** [comment]
here's a simple example gradient accumulation from the `vision_transformer` codebase:

https://github.com/google-research/vision_transformer/blob/ba9a85bdc430daf4da7b9da67b486a4e0f5bb278/vit_jax/train.py#L63-L66

**Comment by AkashGanesan** [comment]
Thanks!! That worked!
